src/pet.js

```diff
@@ -0,0 +1,83 @@
+const MAXIMUM_FITNESS = 10;
+const minHungerLevel = 0;
+
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = minHungerLevel;
+    this.fitness = MAXIMUM_FITNESS;
+    this.children = [];
+};
+
+Pet.prototype = {
+    get isAlive() {
+      return this.age < 30 && this.hunger < 10 && this.fitness > 0;
+    }
+  };
+
+Pet.prototype.growUp = function() {
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive :(');
+      }
+    this.age += 1;
+    this.hunger += 5;
+    this.fitness -= 3;
+  };
```

Comment:
Indentation again here:
```
Pet.prototype.growUp = function() {
    if (!this.isAlive) {
        throw new Error('Your pet is no longer alive :(');
    }
    this.age += 1;
    this.hunger += 5;
    this.fitness -= 3;
};
```

There is quite a lot of this to sort out here, so rather than me manually adding all the corrections it would be much easier for you (and more accurate) for you to run prettier across all your files and have it do the heavy lifting for you! Whether you configure it to use 2 or 4 spaces is up to you, but whichever you pick it should be the same across the whole project